package.json

```diff
@@ -18,18 +18,18 @@
   },
   "devDependencies": {
     "@gltf-transform/core": "^3.7.4",
-    "@rollup/plugin-typescript": "^11.0.0",
-    "@types/jest": "^26.0.15",
-    "@typescript-eslint/eslint-plugin": "^4.6.1",
-    "@typescript-eslint/parser": "^4.6.1",
+    "@rollup/plugin-typescript": "^11.1.6",
+    "@types/jest": "^29.5.12",
+    "@typescript-eslint/eslint-plugin": "^7.9.0",
+    "@typescript-eslint/parser": "^7.9.0",
     "cross-env": "^7.0.2",
     "downlevel-dts": "^0.11.0",
-    "eslint": "^7.12.1",
+    "eslint": "8",
```

Comment:
is the version specifier intentional? eslint seems to bump minor frequently so it might be reasonable enough to do this though

Reply:
ESlint v9 is out but not very well supported by plugins for now. Basically this means latest v8 and I don't think there'll be many v8 releases from now on.

Reply:
sure! if it's intentional it's fine